nextflow.config

```diff
@@ -91,72 +85,181 @@ params {
           "--fastq test_data/barcode01/reads.fastq.gz",
       ]
       agent = null
-      container_sha = "sha156f44a25dd22e39eb10e82d12e0584d41dde909"
+      container_sha = "sha256:4ff33113bd8f0b87ca971ec1e313d8e022b6131829fdd33694d490b399254f9f"
     }
 }
 
+// Load base.config by default for all pipelines
+includeConfig 'conf/base.config'
 
 manifest {
     name            = 'snowy-leopard/scylla'
     author          = 'Snowy Leopard'
     homePage        = 'https://github.com/snowy-leopard/scylla'
     description     = 'Classify metagenomic sequence data from human respiratory infections.'
-    mainScript      = 'ingest.nf'
+    mainScript      = 'main.nf'
     nextflowVersion = '>=20.10.0'
     version         = 'v0.0.1'
 }
 
-
-executor {
```

Comment:
I _think_ this was the bit that was breaking things when we were trying to get this running at ABPHM